flink-end-to-end-tests/test-scripts/common_s3.sh

```diff
@@ -72,4 +75,22 @@ function s3_setup_with_provider {
   set_config_key "$2" "com.amazonaws.auth.EnvironmentVariableCredentialsProvider"
 }
 
-source "$(dirname "$0")"/common_s3_operations.sh
\ No newline at end of file
+###################################
+# Fetches the result in part files into the OUTPUT_PATH.
+#
+# Globals:
+#   OUTPUT_PATH, OUT
+# Arguments:
+#   $1 - single?
+# Returns:
+#   None
+###################################
+function fetch_complete_result {
+  if [[ "$1" == true ]]; then
+    s3_get_by_full_path_and_filename_prefix "$OUTPUT_PATH" "${OUT}" "" false
```

Comment:
We need a better differentiation between `OUTPUT_PATH` and `OUT`, it's impossible to tell at a glance what the difference is.

Reply:
I renamed OUT to S3_PREFIX, it's originated from the original test.